digitalpy/core/component_management/controllers/component_management_general_controller.py

```diff
@@ -1,57 +1,136 @@
 #######################################################
-# 
+#
 # core_name_general_controller.py
 # Python implementation of the Class CoreNameGeneralController
 # Generated by Enterprise Architect
 # Created on:      16-Dec-2022 10:56:05 AM
 # Original author: Giu Platania
-# 
+#
 #######################################################
-from Catalog.Implementation.Libraries.Digitalpy.digitalpy.Async.routing.controller import Controller
+from typing import Any, Dict, List
+from digitalpy.core.main.controller import Controller
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I003 isort expected 1 blank line in imports, found 0